internal/services/servicefabricmanaged/service_fabric_managed_cluster_resource.go

```diff
@@ -202,6 +216,9 @@ func (k ClusterResource) Arguments() map[string]*pluginsdk.Schema {
 			ValidateFunc: validation.IntBetween(1500, 65535),
 		},
 		"lb_rule": lbRulesSchema(),
+
+		"ns_rule": nsRulesSchema(),
```

Comment:
can we change this to network_security_rules

Reply:
> can we change this to network_security_rules

Fixed:).